src/components/MapContent.vue

```diff
@@ -13,35 +12,33 @@
         @onFullscreen="onFullscreen"
         :state="stateToSet"
         ref="flow"
-        @hook:mounted="flowMounted"
+        @vue:mounted="flowMounted"
       />
     </div>
 </template>
 
 <script>
 /* eslint-disable no-alert, no-console */
-import Vue from "vue";
-import SplitFlow from './SplitFlow';
+import SplitFlow from './SplitFlow.vue';
 import EventBus from './EventBus';
-import store from '../store';
+import { mapStores } from 'pinia';
+import { useSettingsStore } from '../stores/settings';
 import { findSpeciesKey } from './scripts/utilities.js';
-import {MapSvgSpriteColor} from '@abi-software/svg-sprite';
+import { MapSvgSpriteColor} from '@abi-software/svg-sprite';
```

Comment:
Is there any affect if not import the SVG style here? From what I met earlier during the migration, it does affect the style.

Reply:
It does affect it if I remember correctly.

Reply:
I run the vuer locally. Style looks not changed with/without the svg style imported. All good, thanks!